README.md

```diff
@@ -31,25 +32,25 @@ omf install https://github.com/jethrokuan/fzf
 
 ## Requirements
 
-* fish version: `>=2.4.0`
-* fzf versions: `>0.11.3`
+- [fish](https://github.com/fish-shell/fish-shell) `>=2.4.0`
+- [fzf](https://github.com/junegunn/fzf) `>0.11.3`
+
+## About the fzf binary
 
-## FZF binary
-This plugin will fail if the `fzf` binary is not detected in your `PATH`.
+This package will fail if the `fzf` binary is not detected in your `PATH`.
 
-The recommended means of installing `fzf` is through your respective
-package managers. 
+See the [fzf documentation](https://github.com/junegunn/fzf#installation) for instructions to install `fzf` on your system.
 
 ## Usage
 
-| Legacy      | New Keybindings   | Remarks                                                       |
-| -------     | ----------------- | ------------------------------------------------------------- |
-| Ctrl-t      | Ctrl-f            | Find a file.                                                  |
-| Ctrl-r      | Ctrl-r            | Search through command history.                               |
-| Alt-c       | Alt-o             | cd into sub-directories (recursively searched).               |
-| Alt-Shift-c | Alt-Shift-o       | cd into sub-directories, including hidden ones.               |
-| Ctrl-o      | Ctrl-o            | Open a file/dir using default editor ($EDITOR)                |
-| Ctrl-g      | Ctrl-g            | Open a file/dir using xdg-open or open command                |
+| Legacy      | New Keybindings | Remarks                                         |
+| ----------- | --------------- | ----------------------------------------------- |
```

Comment:
Table changes are also from my markdown lint. They are mostly about clearing white space and pretty formatting.